meta-facebook/yv4-ff/src/platform/plat_isr.h

```diff
@@ -17,7 +17,16 @@
 #ifndef PLAT_ISR_H
 #define PLAT_ISR_H
 
-void ISR_MB_DC_STAGUS_CHAGNE();
+#include <stdint.h>
+#include "ipmi.h"
+
+// BIC checks for CXL readiness every three seconds, with a maximum of 10 attempts
+#define CXL_READY_RETRY_TIMES 10
+#define CXL_READY_INTERVAL_SECONDS 3
+#define CXL_HEART_BEAT_LABEL "HB0"
+
+void ISR_MB_DC_STATUS_CHAGNE();
```

Comment:
s/CHAGNE/CHANGE/

Reply:
The function name is revised. Please check.
Thanks.